scripts/session/session-logout.sh

```diff
@@ -1,8 +1,14 @@
-#!/bin/sh
+#!/usr/bin/sh
 #
 # name: Log Out
 # icon: system-log-out
 # description: Log out to the login screen
 # keywords: log out logout
 
-gnome-session-quit --logout
+set -eu
+
+if command -v gnome-session-quit >/dev/null; then
```

Comment:
This would trigger if someone had GNOME installed even if they weren't actually in GNOME

Reply:
True, but it's not worse than the status quo in that regard

I'm not entirely sure which environment variables we reliably have access to within these scripts

I did think about checking if XDG_CURRENT_DESKTOP=GNOME but didn't want to risk breaking this for GNOME users if that guess was wrong

I'll install GNOME and see if I can improve this PR, cheers